packages/components/Display.tsx

```diff
@@ -1,11 +1,14 @@
 import { useEffect } from 'react';
 import { Flex, Grid } from '@chakra-ui/react';
 import DigitBox from './DigitBox';
+import CandidatePhotos from './CadidatePhotos';
 
 import { useVotingContext } from '@packages/features/voting-context';
 
 import PartyInfo from './PartyInfo';
 
+const votingScreenColor = '#D9D9D9';
```

Comment:
Adicionar um todo aqui como comentário, na linha nove:

"TODO(Mamao): Move to centralized theme"

Reply:
![Captura de tela de 2022-08-04 10-58-32](https://user-images.githubusercontent.com/73261443/182865719-6bd9a0f2-8cc4-4b6e-b82c-d11ae9c52772.png)
no meu codigo ta desdo ultimo commit